src/adr/AdrEntry.cs

```diff
@@ -172,10 +179,12 @@ where int.TryParse(fileNum, out fileNumOut)
         }
 
         private static string SanitizeFileName(string title)
-        {
-            return title
-                .Replace(' ', '-')
-                .ToLower();
+        {          
+            var filename = StringUtils.FoldToASCII(title.ToLower());
```

Comment:
I'm unconvinced that this is necessary, I would imagine a lot of non English speakers would love to have their non ascii characters in a filename.

Reply:
Mmm... I don't know, I used the Apache Lucene Fold-To-ASCII approach that is quite used for cleaning filenames and to be sure about filenames consistency across different file systems...

http://opendatadpc.maps.arcgis.com/apps/opsdashboard/index.html#/b0c68bce2cce478eaac82fe38d4138b1

Reply:
Ok, I've done some research and I understand you're point. I see that with the `FAT`-likes, the `ext` family and a couple of others we would run into issues.
So we will keep this